packages/inter-protocol/src/vaultFactory/vaultManager.js

```diff
@@ -952,17 +962,58 @@ const finish = ({ state, facets: { helper } }) => {
   });
 };
 
-const behavior = {
-  collateral: collateralBehavior,
-  helper: helperBehavior,
-  manager: managerBehavior,
-  self: selfBehavior,
-};
-
-const makeVaultManagerKit = defineDurableKindMulti(
+const makeVaultManagerKit = defineDurableFarClassKit(
   makeKindHandle('VaultManagerKit'),
+  {
+    collateral: M.interface('collateral', {
+      makeVaultInvitation: M.call().returns(InvitationShape),
+      getSubscriber: M.call().returns(SubscriberShape),
+      getMetrics: M.call().returns(SubscriberShape),
+      getCompoundedInterest: M.call().returns(RatioShape),
+    }),
+    helper: M.interface(
+      'helper',
+      // not exposed so sloppy okay
+      {},
+      { sloppy: true },
+    ),
+    manager: M.interface('manager', {
+      getGovernedParams: M.call().returns(M.remotable()),
+      maxDebtFor: M.call(AmountShape).returns(M.promise()),
```

Comment:
These pattern guards are an externally visible change, yes? They provide security benefits (automatic protection against  scary monster arguments) yes? Feel free to upgrade "refactor" to "fix" on those grounds.